ast/misc.go

```diff
@@ -384,9 +384,10 @@ func (n *ExecuteStmt) Accept(v Visitor) (Node, bool) {
 // See https://dev.mysql.com/doc/refman/5.7/en/commit.html
 type BeginStmt struct {
 	stmtNode
-	Mode     string
-	ReadOnly bool
-	Bound    *TimestampBound
+	Mode                       string
+	ReadOnly                   bool
+	Bound                      *TimestampBound
+	WithoutExternalConsistency bool
```

Comment:
Change the name to be consistent with the statement?

Reply:
The word "external consistency" is used in tidb code. Do you think we should change them as well?

Reply:
Yes, I think so. Let's change to use the more precise word "causal consistency" in TiDB.